spec/cb/clients/cover_letters_spec.rb

```diff
@@ -67,8 +41,12 @@ module Cb
           stub_request(:put, uri).
               with(:body => post_data.to_json, :headers => headers).
               to_return(:status => 500, :body => error_response.to_json)
-          response = Cb::Clients::CoverLetters.create(name: 'name', text: 'text', oauth_token: 'token')
-          expect(response['errors'][0]).to eq(data)
+          begin
+            Cb.cover_letters.create(name: 'name', text: 'text', oauth_token: 'token')
+            expect(false).to eq(true)
```

Comment:
I did it this way because I need to examine what came back from the exception but I wanted to ensure that we fail if it does not raise the exception.  If I did expect{cover_leters.create}.to raise then I couldn't catch the exception